### PR TITLE
Csweaver/packaging

### PR DESCRIPTION
**Pull request by csweaver**

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1,2 @@
+recursive-include server/app/web/templates *
```
Comment (ttung) [inline MANIFEST.in]: My knowledge of this area is pretty sketchy but my understanding is that MANIFEST.in includes these files into your package, but does not actually write them out to the installed space.  However, you have include_package_data=True, which _might_ do what you need.

Can you verify that these are actually getting installed?

File: MANIFEST.in [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+recursive-include server/app/web/static *
```
Comment (ttung) [inline MANIFEST.in]: trailing NL.

File: bin/build-client [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+npm install --prefix client/ client
+npm run  --prefix client build
+mkdir -p server/app/web/static/img
+cp client/build/index.html server/app/web/templates/
+cp -r client/build/static server/app/web/
+
+cp client/build/favicon.png server/app/web/static/img
+cp client/build/service-worker.js server/app/web/static/js/
```
Comment (ttung) [inline bin/build-client]: Trailing NL.

File: example-dataset/data_schema.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
+{
```
Comment (ttung) [inline example-dataset/data_schema.json]: why this change?

File: server/app/__init__.py [hunk 5]
```diff
@@ -1,55 +1 @@
...
-
-app.register_blueprint(webapp.bp)
-app.register_blueprint(resources.blueprint)
-app.register_blueprint(
-    get_swagger_blueprint(docs, "/api/swagger", produces=["application/json"], title="cellxgene rest api",
-                          description="An API connecting ExpressionMatrix2 clustering algorithm to cellxgene"))
-
-
-app.add_url_rule("/", endpoint="index")
+from . import app
```
Comment (ttung) [inline server/app/__init__.py]: trailing NL.

consider lint / flake8 on your repo?

File: server/app/rest_api/rest.py [hunk 6]
```diff
@@ -88,7 +88,7 @@ class InitializeAPI(Resource):
         }
     })
     def get(self):
-        from app import data, REACTIVE_LIMIT
+        from ..app import data, REACTIVE_LIMIT
```
Comment (ttung) [inline server/app/rest_api/rest.py]: for what it's worth, we had a lengthy argument about this in hca-land, and we eventually settled on no relative imports UP.  only relative imports down or sideways.  i think there's some merit in that decision.

File: server/run.py [hunk 7]
```diff
@@ -1,3 +1,3 @@
-from app import app
+from app.app import app
```
Comment (ttung) [inline server/run.py]: i want to say this is not supported in python 3...?

File: example-dataset/data_schema.json (hunk 4 above)
Comment (csweaver) [reply example-dataset/data_schema.json]: I'm adding the example data and schema so that when users install they can launch and play with cellxgene immediately without having to have a dataset ready in the correct format.

File: MANIFEST.in (hunk 1 above)
Comment (csweaver) [reply MANIFEST.in]: Sure, I'll double check that.

File: server/app/__init__.py (hunk 5 above)
Comment (csweaver) [reply server/app/__init__.py]: Is there a linter you recommend for the non-python files? (the ones you gave the trailing newline comment to?)

File: server/app/rest_api/rest.py (hunk 6 above)
Comment (csweaver) [reply server/app/rest_api/rest.py]: I'd be interested in seeing the discussion, do you have a link? I'm happy to make the change, just want to be able to articulate the reasons later.

File: MANIFEST.in (hunk 1 above)
Comment (csweaver) [reply MANIFEST.in]: Yes, these are being installed.

**Comment by csweaver** [comment]
@ttung Thanks! I think I've addressed all of your comments and suggestions, care to take another look?

File: server/run.py (hunk 7 above)
Comment (ttung) [reply server/run.py]: Was kind of curious why this was inconsistent with my experience, so I did a bit of experimentation: top-level script's path injects its directory into sys.path, but other modules/packages do not.
